packages/twenty-server/src/engine/core-modules/auth/token/services/access-token.service.ts

```diff
@@ -42,9 +46,9 @@ export class AccessTokenService {
     const expiresIn = this.environmentService.get('ACCESS_TOKEN_EXPIRES_IN');
 
     if (!expiresIn) {
-      throw new AuthException(
+      throw new EnvironmentException(
```

Comment:
Ideally in this case we should be able to validate its presence earlier with class-validator in the config module (the same way we validate if something is a number, a valid url etc)

Reply:
Actually you can remove this check since there is a default value!